src/data/list/big_operators.lean

```diff
@@ -106,7 +106,7 @@ end
 @[simp, to_additive]
 lemma prod_take_mul_prod_drop :
   ∀ (L : list M) (i : ℕ), (L.take i).prod * (L.drop i).prod = L.prod
-| [] i := by simp
+| [] i := by simp [@zero_le' ℕ]
```

Comment:
These `@zero_le' ℕ` are somewhat ugly. Can you fix that regression please?

Reply:
Not sure how... This is a bug in `to_additive` that shows only now because of the import changes.

Reply:
this has come up before in #10767